src/lib.rs

```diff
@@ -141,13 +159,19 @@ impl Expected for Crv {
 }
 
 #[derive(Clone, Debug, Eq, PartialEq, Serialize, Deserialize)]
-#[serde(try_from = "RawPublicKey")]
+#[serde(try_from = "RawEcPublicKey")]
```

Comment:
Doesn’t this mean that the MLDSA variants cannot be deserialized?

Reply:
We renamed `RawPublicKey` to `RawEcPublicKey` (RawElipticCurvePublicKey) to differentiate it from the new `RawMldsaPublicKey`.

Deserialization for MLDSA is done in the macro, [here](https://github.com/sandbox-quantum/cosey/blob/934ba2511342fd466b6e4af87a4068f9e6b0aece/src/lib.rs#L607-L693).

Reply:
There are two ways to serialize and deserialize data – using the specific structs like `Mldsa44PublicKey`, or using the `PublicKey` enum.  With this implementation, you could serialize a `PublicKey::Mldsa44(_)` or a `Mldsa44PublicKey`, but you could only deserialize the result into a `Mldsa44PublicKey`, not into a `PublicKey`.